src/features/events/components/EventItemComponent/index.tsx

```diff
@@ -18,26 +20,18 @@ import {
   // StyledButton,
 } from './styled'
 
-import { EventActionButton } from '../../../EventActionButton'
-
 export type Props = {
   view: ViewType
   event: ArticleType
-  isLoggedInUserAttending: boolean
-  isLoggedInUserOwner: boolean
```

Comment:
Suddenly, by moving some responsibilities around, we reduced heavily the props waterfall.